doc/src/site/sphinx/install/install_and_start.rst

```diff
@@ -0,0 +1,205 @@
+Installing and Starting
+=======================
+
+This section describes how to download and run Sparkling Water in different environments. Refer to the :ref:`pysparkling` and :ref:`rsparkling` sections for instructions on installing and running PySparkling and RSparkling. 
+
+Download and Run Locally
+------------------------
+
+This section describes how to quickly get started with Sparkling Water on your personal computer (in Spark's ``local`` cluster mode).
+
+1. Download and install Spark (if not already installed) from the `Spark Downloads page <https://spark.apache.org/downloads.html>`__.
+
+  ::
+
+    Choose Spark release: SUBST_SPARK_VERSION
+    Choose package type: Pre-built for Hadoop 2.7 and later
+
+2. Point SPARK_HOME to the existing installation of Spark and export variable MASTER.
+
+  ::
+
+    export SPARK_HOME="/path/to/spark/installation"
+    # To launch a local Spark cluster.
+    export MASTER="local[*]"
+
+3. From your terminal, run:
+
+  ::
+
+    cd ~/Downloads
+    unzip sparkling-water-SUBST_PROJECT_VERSION.zip
+    cd sparkling-water-SUBST_PROJECT_VERSION
+    bin/sparkling-shell --conf "spark.executor.memory=1g"
+
+4. Create an H2O cloud inside the Spark cluster:
+
+  ::
+
+    import org.apache.spark.h2o._
+    val h2oContext = H2OContext.getOrCreate(spark)
+    import h2oContext._ <br />
+
+5. Begin using Sparkling Water by following `this demo <https://github.com/h2oai/sparkling-water/tree/master/examples#step-by-step-weather-data-example>`__, which imports airlines and weather data and runs predictions on delays.
+
+
+Run on Hadoop
+-------------
+
+This section describes how to launch Sparkling Water on Hadoop using YARN.
+
+**Note**: The following is a list of supported Hadoop distributions: SUBST_H2O_DRIVERS_LIST
+
+1. Download Spark (if not already installed) from the `Spark Downloads page <https://spark.apache.org/downloads.html>`__.
+
+  ::
+
+    Choose Spark release: SUBST_SPARK_VERSION
+    Choose package type: Pre-built for Hadoop 2.7 and later
+
+2. Point SPARK_HOME to the existing installation of Spark.
+
+  ::
+
+    export SPARK_HOME='/path/to/spark/installation'
+
+3. Set the HADOOP_CONF_DIR and Spark MASTER environmental variables.
+
+  ::
+
+    export HADOOP_CONF_DIR=/etc/hadoop/conf
+    export MASTER="yarn"
+
+4. Download Spark and use ``sparkling-shell`` to launch Sparkling Shell on YARN.
+
+  ::
+
+    wget <script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>/sparkling-water-SUBST_PROJECT_VERSION.zip 
```

Comment:
Seems like this should not be here `wget <script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>/sparkling-water-SUBST_PROJECT_VERSION.zip`

It's enough just to say download Spark

Reply:
Should that also be removed from the download page?

Reply:
on download page it resolves correctly to  `wget https://s3.amazonaws.com/h2o-release/sparkling-water/rel-2.2/28/sparkling-water-2.2.28.zip`

I would at the end keep it here, but make sure it is ` wget https://s3.amazonaws.com/h2o-release/sparkling-water/rel-2.2/28/sparkling-water-2.2.28.zip ` with the right substitutions

Reply:
Hmm... I copied it directly from the download page (with the same substitutions).

Reply:
But it should not contain the `scripts` blocks etc, should be just:

wget https://s3.amazonaws.com/h2o-release/sparkling-water/rel-SUBST_SW_MAJOR_VERSION/SUBST_SW_MINOR_VERSION/sparkling-water-SUBST_SW_VERSION.zip

What do you think please?